client/analysisui/layoutactionmanager.js

```diff
@@ -44,32 +44,188 @@ const LayoutActionManager = function(view) {
         }
     };
 
-    this.bindActionParams = function(sourceName, target, targetProperty, isCompare, compareValue) {
+    this.bindActionParams = function(target, targetProperty, bindData) {
         return {
-            onChange: sourceName,
+            onChange: bindData.sourceNames,
             execute: (ui) => {
-                let value = ui[sourceName].value();
-                if (isCompare)
-                    value = value === compareValue;
+                let value = bindData.bindFunction(ui);
+                if (bindData.inverted)
+                    value = !value;
                 target.setPropertyValue(targetProperty, value);
             }
         };
     };
 
+    //enable: ~((fred:(tom)||fred:pop||~(tony:touch))&&steve:cat)
```

Comment:
otherwise i treat it as a literal. might be worth adopting this in the .u.yaml too.

granted, for `enable` it's almost always going to be an expression, but there might be other properties/bindings we do, where it would be good to disambiguate them